cmd/sebak/common/util.go

```diff
@@ -23,6 +23,16 @@ func PrintFlagsError(cmd *cobra.Command, flagName string, err error) {
 	os.Exit(1)
 }
 
+func PrintError(cmd *cobra.Command, err error) {
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n\n", err)
+	}
+
+	cmd.Help()
+
+	os.Exit(1)
+}
```

Comment:
This is not used anywhere (and generally we should not `os.Exit` like this)

Reply:
`PrintError` is used for https://github.com/bosnet/sebak/pull/511/files/efe9f927550b50897f013a48243b352ecf2bafd3#diff-f7927137f9b4e201f9f6cc347f0b0e1dR324 and this is another expression of `PrintFlagsError()`. Anyway, it should be removed because `node.NewLocalNode()` does not return `error` :)